src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -140,12 +140,22 @@ protected override void ProcessRecord()
                 }
                 catch (FileNotFoundException ex)
                 {
-                    ErrorRecord errorRecord = new ErrorRecord(ex,
+                    ErrorRecord errorRecord = new ErrorRecord(
+                        ex,
                         "FileNotFound",
                         ErrorCategory.ObjectNotFound,
                         path);
                     WriteError(errorRecord);
                 }
+                catch (UnauthorizedAccessException ex)
+                {
+                    ErrorRecord errorRecord = new ErrorRecord(
```

Comment:
Similar thing here too.
```suggestion
                    var errorRecord = new ErrorRecord(
```